addons/image-uploader/addon.json

```diff
@@ -1,15 +1,23 @@
 {
-  "name": "Forums image uploader",
-  "description": "Integrates image uploading into the forums.",
+  "name": "Image uploader",
```

Comment:
maybe have it be Comment & forums image uploader?

Reply:
Too long

Reply:
that would change it to a phrase (#2764), but that would only be 32 chars, or 11 more. There are definitley longer ones. But #2764 might mean that it shouldn't be done?